extensions/bitwarden/src/search.tsx

```diff
@@ -143,13 +160,28 @@ export function ItemList(props: { api: Bitwarden }) {
             title="No matching items found."
             description="Hit the refresh button to sync your vault."
             actions={
-              <ActionPanel>
-                <Action icon={Icon.ArrowClockwise} title={"Refresh Items"} onAction={refreshItems} />
-              </ActionPanel>
+              !state.isLoading && (
+                <ActionPanel>
+                  <VaultActions refreshItems={refreshItems} lockVault={lockVault} logoutVault={logoutVault} />
+                </ActionPanel>
+              )
             }
           />
         </Fragment>
```

Comment:
I think the fragment is no longer necessary here.

Reply:
The empty state is still in the same part of the tenary statement here I think

Reply:
I don't think we need the ternary statement here. The List.EmptyView is supposed to be next to the list items. I agree that the API is confusing.

The <Fragment> is also unecessary.

Reply:
I did this for when the list of items is empty so a different empty message could be shown. Initially I was going to put different actions in the empty states which is why I kept it (and hence the Fragment was required in the true clause).

I'll move the `if` out of the return in my next commit! Cheers.